### PR TITLE
kuanchou/2402 make pv axes order and pv keep as preference

### LINKED ISSUE #2402: [enhancement] make PV diagram axes order as a preference
Currently, x: spatial - y: spectral is the default setting. Although the axes can be switched, some users request to change the default axes order. We could make this axes order to be a preference.

### PR DESCRIPTION
**Pull request by loveluthien**
**Description**

Close #2402. Now, the PV axis order becomes sticky.

**Checklist**

The PV axis order option.

For linked issues (if there are):
- [x] assignee and label added
- [x] GitHub Project estimate added

For the pull request:
- [x] reviewers and assignee added
- [x] GitHub Project estimate added
- [ ] ~changelog updated~ / no changelog update needed
- [x] ~unit test added (for functions with no dependenies)~
- [x] ~API documentation added (for public variables and methods in stores)~

For dependencies:
- [x] e2e test passing / corresponding fix added / ~new e2e test created~
- [x] ~protobuf version bumped~ / no protobuf version bumped needed
- [x] ~protobuf updated to the latest dev commit~ / no protobuf update needed
- [x] ~corresponding ICD test fix added (`BackendService` changed)~ / no ICD test fix needed (`BackendService` unchanged)
- [x] ~user manual prepared (for large new features)~

### CODE REVIEW

File: src/stores/PreferenceStore/PreferenceStore.ts [hunk 1]
```diff
@@ -255,6 +258,10 @@ const DEFAULTS = {
         pvPreviewCubeSizeLimit: 1,
         pvPreviewCubeSizeLimitUnit: MemoryUnit.GB
     },
+    PV: {
```
Comment (YuHsuan-Hwang) [inline src/stores/PreferenceStore/PreferenceStore.ts]: We usually add to `SILENT` for preferences not listed in the dialog and reset them with `resetSilentSettings`.

**Review by YuHsuan-Hwang — COMMENTED** [review]
Looks good. 👍

File: src/stores/PreferenceStore/PreferenceStore.ts (hunk 1 above)
Comment (loveluthien) [reply src/stores/PreferenceStore/PreferenceStore.ts]: Thanks for letting me know this 😄. The code is modified accordingly.

**Review by YuHsuan-Hwang — APPROVED** [review]

**Review by kswang1029 — APPROVED** [review]
looks good. due to the behaviour change, relevant e2e tests are fixed accordingly.

**Comment by YuHsuan-Hwang** [comment]
@loveluthien I prefer we add a change log since the UI behavior is changed.

**Comment by loveluthien** [comment]
@YuHsuan-Hwang done.

**Pull request merged**